tests/human/human.fna

```diff
@@ -0,0 +1,79 @@
+>NC_001422.1 Escherichia phage phiX174, complete genome
```

Comment:
It's just a contig from phiX to have a fake human.fna, we can rename the contig if necessary